src/classes/Toggle.js

```diff
@@ -0,0 +1,301 @@
+import AbstractDomElement from '../AbstractDomElement'
```

Comment:
Codacy has a fix for the issue: [Insert `import·{·disableBodyScroll,·enableBodyScroll·}·from·'body-scroll-lock'⏎`](https://app.codacy.com/gh/BeAPI/be-a11y/pullRequest?prid=8001873)
```suggestion
import { disableBodyScroll, enableBodyScroll } from 'body-scroll-lock'
```